lib/tests/convert.test.js

```diff
@@ -34,10 +34,11 @@ describe('#convert', () => {
     expect(parsed[1].text).to.equal('Can’t see original dreams');
   });
 
-  it('should catch non numerical indices', () => {
-    const { status } = convert(badIndexNotNumerical, '.srt', '.srt');
-    expect(status.invalidEntries).to.not.be.empty;
-  });
+  // TODO!
```

Comment:
Is this TODO for later or no?

Reply:
I would like to look into this in the future.  I can delete and make a note/ticket for it, or leave here as a reminder.  What works best?

Reply:
Removed the unused test and created a ticket to investigate later.